lightning-network-book.tex

```diff
@@ -100,16 +100,16 @@
   description={}}
 \newglossaryentry{script}{name={Bitcoin Script},
   description={}}
-\newglossaryentry{msat}{name={millisatoshi},
+\newglossaryentry{msat}{name={Millisatoshi},
 description={}}
 
 
   \newglossaryentry{rsmc}{name={Revocable Sequence Maturity Contract},
-    description={This \gls{contract} is used to contract a payment channel between two bitcoin users who do not need to trust each other. The name comes from a sequence of states which are encoded as commitment transactions and can be revoked if wrongfully published and mined by the bitcoin netowork.}}
+    description={This \gls{contract} is used to construct a payment channel between two Bitcoin users who do not need to trust each other. The name comes from a sequence of states which are encoded as commitment transactions and can be revoked if wrongfully published and mined by the Bitcoin netowork. These contracts are commonly referred to as RSMCs.}}
```

Comment:
Does that last Scentence actually make sense? I am defining the RSMCs so it seems a little bit redundant

Reply:
I will merge this now but modify it later. I will put abbreviations to the beginning of each glossary term. (also in the text they should be included when the term is defined the first time. Not clear where this will be)